tensorflow_serving/model_servers/main.cc

```diff
@@ -176,10 +178,27 @@ int DeadlineToTimeoutMillis(const gpr_timespec deadline) {
 
 class PredictionServiceImpl final : public PredictionService::Service {
  public:
-  explicit PredictionServiceImpl(ServerCore* core, bool use_saved_model)
+  explicit PredictionServiceImpl(ServerCore* core,
+		                 bool use_saved_model,
+				 const string& model_config_file)
       : core_(core),
         predictor_(new TensorflowPredictor(use_saved_model)),
-        use_saved_model_(use_saved_model) {}
+        use_saved_model_(use_saved_model) {
+    if (!model_config_file.empty()) {
```

Comment:
I'm pretty sure we'll want a flag to control whether to do this dynamic config reloading. We'll also want to be able to control the polling period. Maybe we can combine these into a single flag: config_reload_polling_period_seconds /* setting to 0 disables dynamic config reloading */. And have it default to 0 (no polling/reloading). WDYT?